src/apiMocks/mockData.ts

```diff
@@ -0,0 +1,202 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { faker } from "@faker-js/faker";
+import { OneRepScanResult } from "../customTypes";
+
+// This is a full list of scan results, all pages, and would normally be
+// stored in the `onerep_scan_results.onerep_scan_results` column
+// as `jsonb`.
+export const mockedOneRepScanResult = {
+  data: [
+    {
+      id: 3038059606048768,
+      age: 3392703368790016,
```

Comment:
Unclear on this is a "real" mock `age` or a random number, but&hellip;

Does OneRep not have public API docs?
I found https://onerep.com/api but it seems to want me to submit a request to OneRep to see if I "qualify" to read their API docs.

I had assumed from the schema below that `age` would have been my age in years. But even if it's milliseconds that isn't making sense.

If that is a JavaScript date in ms, it looks like it's from the year 109480, which seems implausible. Even if i divide by 1000 and reparse, it thinks the date is 2077. So I must be totally misunderstanding what this `age` is if it isn't my age or possibly milliseconds.
```js
new Date(3392703368790016).toLocaleDateString() // '6/27/109480'
new Date(3392703368790016/1000).toLocaleDateString() // '7/5/2077'
```

Oh, unless it's nanoseconds:

```js
new Date(3392703368790016 / 1000 / 1000).toLocaleDateString() // '2/8/1970'
new Date(3392703368790016 / 1000 / 1000).getTime() // 3_392_703_368

const MS_IN_DAY = 1000 * 60 * 60 * 24; // 86_400_000
console.log(3_392_703_368 / 86_400_000); // 39.27 days
```